core/modules/SConscript

```diff
@@ -1,35 +1,159 @@
 # -*- python -*-
 # Qserv core/modules SConscript
 #
-# Setup our environment
-#
-Import('*')
-import glob, os, re, sys
-import state
-from itertools import chain, izip, repeat
+
+"""
+This SConscript file defines all targets produced from code in core/modules.
+
+This file is loaded from top-level SConstruct and its main purpose is to
+return tuple with two items - install targets and test targets - former is
+assigned to "install" alias, latter to "test" alias.
+
+The code in core/modules is split into a bunch of modules, modules do most
+of the job definig targets for their own code, this includes:
+  - making shared object (*.os) from C++ code
```

Comment:
No, it's confusing but correct :) scons compiles code into *.o or *.os depending on whether it's static/shared object. Shared objects are normally collected into shared libs *.so (which are also called shared objects).